packages/core-ethereum/src/types/ticket.spec.ts

```diff
@@ -74,20 +75,20 @@ describe('test ticket construction', function () {
 
     assert(ticket.counterparty.eq(ticketData.counterparty), 'wrong counterparty')
     assert(ticket.challenge.eq(ticketData.challenge), 'wrong challenge')
-    assert(ticket.epoch.eq(ticketData.epoch), 'wrong epoch')
+    assert(ticket.epoch.toBN().eq(ticketData.epoch.toBN()), 'wrong epoch')
     assert(ticket.amount.toBN().eq(ticketData.amount.toBN()), 'wrong amount')
     assert(ticket.winProb.eq(ticketData.winProb), 'wrong winProb')
-    assert(ticket.channelIteration.eq(ticketData.channelIteration), 'wrong channelIteration')
+    assert(ticket.channelIteration.toBN().eq(ticketData.channelIteration.toBN()), 'wrong channelIteration')
   })
 
   it('should generate the hash correctly #1', async function () {
-    const expectedHash = new Hash(stringToU8a('0x6db4ce6e1254d1b3137c98570be35428f0764fe383cc7242a7c9cd1cc11f81d2'))
+    const expectedHash = new Hash(stringToU8a('0x4d5137ffaad9d5eb8d3cd6252fd8e7fc9b04d24e7f3cedf88d21f569d5a57c86'))
```

Comment:
Yeah, changes like this made me think we indeed need a place to put these addresses and numbers. Can be done in a separate PR tho, not a big deal.